core/src/cats/sql_update.cc

```diff
@@ -61,7 +61,7 @@ bool BareosDb::AddDigestToFileRecord(JobControlRecord* jcr,
   EscapeString(jcr, esc_name, digest, len);
   Mmsg(cmd, "UPDATE File SET MD5='%s' WHERE FileId=%s", esc_name,
        edit_int64(FileId, ed1));
-  retval = UPDATE_DB(jcr, cmd);
+  retval = UPDATE_DB(jcr, cmd) > 0 ? true : false;
```

Comment:
`x > 0` already is a boolean expression, no need to convert it again.
```suggestion
  retval = UPDATE_DB(jcr, cmd) > 0;
```